lib/rules/quote-props.js

```diff
@@ -68,5 +97,14 @@ module.exports = function(context) {
 module.exports.schema = [
     {
         "enum": ["always", "as-needed"]
+    },
+    {
+        "type": "object",
```

Comment:
I think this has to be tweaked to work correctly when the second item is missing. @btmills any advice here?